### PR TITLE
login functionality + sql injection protection function

### PR DESCRIPTION
**Pull request by ivailo-devrix**

### CODE REVIEW

File: includes/theme-compat/header.php [hunk 1]
```diff
@@ -1,4 +1,12 @@
 <?php
+// server should keep session data for AT LEAST 1 hour
+ini_set('session.gc_maxlifetime', 3600);
+
+// each client should remember their session id for EXACTLY 1 hour
+session_set_cookie_params(3600);
```
Comment (Neychok) [inline includes/theme-compat/header.php:6]: I think those should be in the `config.php` file

**Review by Neychok — CHANGES_REQUESTED** [review]
Just one small thing and we can merge this.

File: includes/theme-compat/header.php (hunk 1 above)
Comment (ivailo-devrix) [reply includes/theme-compat/header.php:6]: This is currently a test functionality. I've moved it to the mandatory includes file "required-includes.php" that loads on every page. 

In the config file, my idea is to have only and only constants. My idea is to keep it as clean as possible and to have absolutely no functionality in it.

**Review by Neychok — APPROVED** [review]
Let's get this puppy going.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 .idea/
 .work/
 img/company/
+test.php
```
Comment (metodiew) [inline .gitignore:4]: @yanachkov-devix what would happen if you happen to have a `test.php` file in your Unit tests in the future?

File: includes/db-connect.php [hunk 3]
```diff
@@ -21,15 +22,38 @@ function open_db_conn()
 }
 
 
+//Preventing SQL Injection Attacks in Postgres
```
Comment (metodiew) [inline includes/db-connect.php:25]: I did not know we are using Postgres heres🙈

**Pull request merged**